tests/clock/deactivateRooms.js

```diff
@@ -72,15 +72,20 @@ describe('tests/clock/deactivateRooms.js >', () => {
     .then((room) => {
       testEvent.roomId = room.id;
       testEvent.createdAt = MOON_LANDING;
-      return Event.create(testEvent);
+      return (Event.create(testEvent));
     })
     .then((event) => {
       deactivateRooms.execute()
       .then((deactivatedRooms) => {
-        Room.findAll({ where: { active: true } })
+        Room.findOne({ where: { id: event.roomId } })
         .then((r) => {
+          /**
```

Comment:
the convention for multiline comments *inside* a function is just one asterisk to open

```
/*
 * my multiline comment
 * goes here
 */
```

You only use two asterisks for module- and function-level multiline comments.

... and thanks for adding the comment here--very helpful!